src/readquartetdata.jl

```diff
@@ -1044,3 +1044,223 @@ function createQuartet(taxa::Union{Vector{<:AbstractString},Vector{Int}}, qvec::
     end
     return Quartet(num,names,[1.0,0.0,0.0])
 end
+
```

Comment:
This function might make sense in `PhyloNetworks`. Though, if it remains here, we should get rid of camel casing, perhaps `readnexus_treeblocks`